freature/credits-choapi-adobe (#2075)
mercadopago/devsite-docs: guides/adobe-commerce/checkout-api.en.md

----[mlb]----

| Payment Method | Description |
|---|---|
| Cards | Allows integration with credit and debit cards |

> | 2 Cards | Allows integration with 2 credit and debit cards |

Author: ```suggestion
| 2 Cards | Allows integration with 2 cards, wether they are credit or debit cards. |
```

> | 2 Cards | Allows integration with 2 cards, wether they are credit or debit cards. |

Reviewer: ```suggestion
| 2 Cards | Allows integration with 2 cards, whether they are credit or debit cards. |
```

> | Pix | Bank transfer available 24 hours a day. The Pix payment option will only be displayed if a Pix key is registered in Mercado Pago. If you haven't created one yet, click [here](https://www.youtube.com/watch?v=60tApKYVnkA) and see the step-by-step. |

Reviewer: ```suggestion
| Pix | Bank transfer available 24 hours a day. The Pix payment option will only be displayed if a Pix key is registered in Mercado Pago. If you haven't created one yet, [see the step-by-step](https://www.youtube.com/watch?v=60tApKYVnkA). |
```

Reviewer: Dejo esta sugerencia para hacer que el link sea parte del contenido!

| Offline methods | Payments via boleto bancário and lottery house. |
| Parcelamento via Pix em até 12x | Allows the customer the option to finance a purchase without a card, while the seller receives the total amount of the money thanks to this Mercado Pago administrated credit line. |

------------

----[mlm]----

| Payment Method | Description |
|---|---|
| Cards | Allows integration with credit and debit cards |

> | 2 Cards | Allows integration with 2 credit and debit cards |

Author: ```suggestion
| 2 Cards | Allows integration with 2 cards, wether they are credit or debit cards. |
```

> | 2 Cards | Allows integration with 2 cards, wether they are credit or debit cards. |

Reviewer: ```suggestion
| 2 Cards | Allows integration with 2 cards, whether they are credit or debit cards. |
```

| Offline methods | Paycash |
| Up to 12 installments without a card with Mercado Pago | Allows the customer the option to finance a purchase without a card, while the seller receives the total amount of the money thanks to this Mercado Pago administrated credit line. |

------------

----[mla]----

| Payment Method | Description |
|---|---|
| Cards | Allows integration with credit and debit cards |

> | 2 Cards | Allows integration with 2 credit and debit cards |

Author: ```suggestion
| 2 Cards | Allows integration with 2 cards, wether they are credit or debit cards.|
```

> | 2 Cards | Allows integration with 2 cards, wether they are credit or debit cards.|

Author: ```suggestion
| 2 Cards | Allows integration with 2 cards, whether they are credit or debit cards.|
```
